examples/basic-otlp/src/main.rs

```diff
@@ -23,18 +23,38 @@ fn init_tracer() -> Result<sdktrace::Tracer, TraceError> {
 
 // Skip first immediate tick from tokio, not needed for async_std.
 fn delayed_interval(duration: Duration) -> impl Stream<Item = tokio::time::Instant> {
-    opentelemetry::util::tokio_interval_stream(duration).skip(1)
+   opentelemetry::util::tokio_interval_stream(duration).skip(1)
+}
+
+use opentelemetry::metrics::Descriptor;
+use opentelemetry::sdk::export::metrics::Aggregator;
+use opentelemetry::sdk::export::metrics::AggregatorSelector;
+use std::sync::Arc;
+
+#[derive(Debug)]
+struct NOPAggregator();
```

Comment:
Style nits: this should be named `NopAggregator` (maybe `NoopAggregator` is even easier to understand for most readers?) and you should just leave off the `()`.

Reply:
Yeah, I think we most used `Noop[Component Name]` to name the no-op components.

Reply:
i think i will keep this example as is and add another example with a custom aggregator selector.

Reply:
removed altogether